src/applications/vaos/services/slot/transformers.js

```diff
@@ -1,26 +1,6 @@
-import moment from '../../utils/moment-tz';
-import { getTimezoneBySystemId } from '../../utils/timezone';
+import moment from 'moment';
 import { FREE_BUSY_TYPES } from '../../utils/constants';
 
-/**
- *
- * The slot datetimes we get back from VistA include
- * an offset of +00:00 that isn't actually accurate. The times returned are
- * already in the time zone of the facility. In order to prevent
- * moment from using this offset, we'll remove it initially, apply the correct
- * timezone and then convert to UTC which is what FHIR uses
- *
- * @param {string} dateTime A VistA slot datetime
- * @param {string} timezone An IANA timezone, e.g. "America/Denver"
- * @returns {string} A VistA time slot formatted to UTC
- */
-function convertToUTC(dateTime, timezone) {
```

Comment:
I took this out because we would need to map this back to the facility timezone before showing the slots and before sending it in the final POST. We might need to do that in the future depending on what times VSP sends us, but I don't want to do that work until we know we need to.